README.md

```diff
@@ -30,12 +34,8 @@ Parameter|Value|Description
 ---|---|---
 `fastq1`|File|Read 1 input fastq file
 `fastq2`|File|Read 2 input fastq file
-`referenceGenome`|String|reference genome to use
-`referenceGenomeIndex`|String|.fai file for the respective ref. genome fasta file
-`referenceModule`|String|module to load the reference genome
+`reference`|String|reference id for using with the analysis
```

Comment:
should we include which genome builds are supported?

Reply:
Normally we would check this in the olive, at least I do not see it used ]consistently] in the workflows